### ISSUE TITLE
timestamp slider(Not really an issue more of a request)

### ISSUE THREAD

**Issue by sulfurexcretion** [nn9dev/stemplayerplayer#5]
its sort of inconvenient to hear a part but you have to restart the song to get back to it so I was wondering if that was a possibility.

**Comment by nn9dev** [comment]
No, I absolutely agree. Only issue with this is that I'd have to rewrite the sound system. Since pygame's mixer is limited in this use case. I was looking for a few different alternatives last night, but couldn't really find many that would have all the appropriate functions (ability to play multiple streams at once, seek/tracking of time passed, ability to start playback at a specific point). SDL2 may have what I need, but I can't make many promises. If you just so happen to know any that would fit that bill, please let me know. 

TL;DR
I'll look into it :)
